examples/react_webpack_hotreload/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@npm//webpack-dev-server:index.bzl", server = "webpack_dev_server")
+
+server(
```

Comment:
any particular reason to rename the symbol on load instead of keeping "webpack_dev_server"?

Reply:
no, i sliced this up from my working project where i was replacing http_server